pycbc/events/stat.py

```diff
@@ -1277,7 +1277,8 @@ class ExpFitSGFgBgNormStatistic(PhaseTDStatistic,
     Statistic combining PhaseTD, ExpFitSGBg and additional foreground info.
     """
 
-    def __init__(self, sngl_ranking, files=None, ifos=None, **kwargs):
+    def __init__(self, sngl_ranking, files=None, ifos=None,
+                 reference_ifos='H1,L1', **kwargs):
```

Comment:
Data-structure nitpick: I would have chosen an iterable of strings for `reference_ifos` (probably a set) instead of a comma-separated string.

Reply:
This has to come through a specific command-line structure (the stat's CLI doesn't know what kwargs might be given) where adding support for lists would probably be more trouble than it's worth.